server/data/quotes.json

```diff
@@ -2,6 +2,10 @@
   {
     "author": "Abhinav Dubey",
     "sentence": "Language is a weapon to know the knowledge of a person"
+  },
+    {
+    "author": "Abhra Kanti Dubey",
+    "sentence": "Language is a Feeling not just normal words"
```

Comment:
Hey! Please fix indentation of the curly brace, add a period to the end of the sentence, and make feeling in lowercase.